<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtColorize::Internal::QtColorizeCoreOptionsWidget</class>
 <widget class="QWidget" name="QtColorize::Internal::QtColorizeCoreOptionsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QScrollArea" name="detailsScrollArea">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QAbstractScrollArea::AdjustToContents</enum>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>212</width>
        <height>272</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <property name="sizeConstraint">
        <enum>QLayout::SetFixedSize</enum>
       </property>
       <property name="leftMargin">
        <number>6</number>
       </property>
       <property name="topMargin">
        <number>6</number>
       </property>
       <property name="rightMargin">
        <number>6</number>
       </property>
       <property name="bottomMargin">
        <number>6</number>
       </property>
       <item row="4" column="2">
        <widget class="QToolButton" name="redResetToolButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip">
          <string>Reset red color</string>
         </property>
         <property name="arrowType">
          <enum>Qt::LeftArrow</enum>
         </property>
        </widget>
       </item>
       <item row="7" column="0" colspan="3">
        <spacer name="spacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>200</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="4" column="1">
        <widget class="QToolButton" name="redToolButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QToolButton" name="greenResetToolButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip">
          <string>Reset green color</string>
         </property>
         <property name="arrowType">
          <enum>Qt::LeftArrow</enum>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QToolButton" name="greenToolButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="greenLabel">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Green:</string>
         </property>
         <property name="buddy">
          <cstring>greenToolButton</cstring>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="redLabel">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Red:</string>
         </property>
         <property name="buddy">
          <cstring>redToolButton</cstring>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="alignmentLabel">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Alignment:</string>
         </property>
         <property name="buddy">
          <cstring>alignmentSpinBox</cstring>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QSpinBox" name="alignmentSpinBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip">
          <string>Align comments by specified value. Value 0 means alignment disabled</string>
         </property>
         <property name="maximum">
          <number>300</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>greenToolButton</tabstop>
  <tabstop>greenResetToolButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
